src/hooks/useCart/useCart.tsx

```diff
@@ -209,6 +214,13 @@ export const useCart = (
           ) {
             numUnverifiedTransactions += 1;
           }
+
+          const inputValue = identifiers.map(identifier => identifier.value);
+          inputValues = !inputValue.includes("")
+            ? inputValues.concat(inputValue)
+            : inputValues;
+          uniqueInput = validateUniqueInputs(inputValues) ? true : false;
```

Comment:
hmm, since it's a boolean, would renaming it as `hasUniqueInputs` make it clearer? What do you think haha